app/src/main/res/layout/activity_event_initial.xml

```diff
@@ -224,8 +224,8 @@
                         android:orientation="vertical" />
 
                     <include
-                        android:id="@+id/geometry"
```

Comment:
Related to the previous comment. Here we are showing the coordinates field in event inital as it should happen in landscape.